examples/fetch_playlist.rs

```diff
@@ -0,0 +1,10 @@
+use listenbrainz::raw::Client;
+
+fn main() {
+    let client = Client::new();
+
+    let playlist = client
+        .get_playlist("f316bb0e-8e26-44f8-a802-6d2a3688fc7d")
```

Comment:
Imo, probably better to take a CLI argument (this is what other examples do) than hardcode a playlist that might disappear. Not a dealbreaker for me though.